frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -177,6 +178,134 @@ export function ReplayAuthorizedDomains(): JSX.Element {
     )
 }
 
+export function ReplaySummarySettings(): JSX.Element | null {
+    const { updateCurrentTeam } = useActions(teamLogic)
+
+    const { currentTeam } = useValues(teamLogic)
+
+    if (!currentTeam) {
+        return null
+    }
+
+    const defaultConfig = {
+        opt_in: false,
+        preferred_events: [],
+        excluded_events: ['$feature_flag_called'],
+        included_event_properties: ['elements_chain', '$window_id', '$current_url', '$event_type'],
+    }
+    const currentConfig: SessionRecordingSummaryConfig = currentTeam.session_recording_summary_config || defaultConfig
+
+    return (
+        <div className="flex flex-col gap-2">
+            <div>
```

Comment:
Possibly just in the screenshot but looks like this is clipping the left hand border of the button

Reply:
Not the case